cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/internal/documentstore/ForceInsertItem.java

```diff
@@ -30,7 +30,7 @@
 
     /**
      * @param rev                   A {@code DocumentRevision} containing the information for a
-     *                              revision from a remote datastore.
+     *                              revision from a remote DocumentStore.
```

Comment:
"remote datastore" in this context is meant to mean a couch or cloudant instance (or anything which supports the couch replication protocol)